posthog/models/person/sql.py

```diff
@@ -228,6 +229,63 @@
     database=CLICKHOUSE_DATABASE,
 )
 
+#
+# person_distinct_id_overrides: This table contains rows for all (team_id,
+# distinct_id) pairs where the person_id has changed and those updates have not
+# yet been integrated back into the events table via squashing.
+#
+
+PERSON_DISTINCT_ID_OVERRIDES_TABLE = "person_distinct_id_overrides"
+
+PERSON_DISTINCT_ID_OVERRIDES_TABLE_BASE_SQL = PERSON_DISTINCT_ID2_TABLE_BASE_SQL
+
+PERSON_DISTINCT_ID_OVERRIDES_TABLE_ENGINE = lambda: ReplacingMergeTree(
+    PERSON_DISTINCT_ID_OVERRIDES_TABLE, ver="version"
+)
+
+PERSON_DISTINCT_ID_OVERRIDES_TABLE_SQL = lambda: (
+    PERSON_DISTINCT_ID_OVERRIDES_TABLE_BASE_SQL
+    + """
+    ORDER BY (team_id, distinct_id)
+    SETTINGS index_granularity = 512
+    """
+).format(
+    table_name=PERSON_DISTINCT_ID_OVERRIDES_TABLE,
+    cluster=CLICKHOUSE_CLUSTER,
+    engine=PERSON_DISTINCT_ID_OVERRIDES_TABLE_ENGINE(),
+    extra_fields=f"""
+    {KAFKA_COLUMNS_WITH_PARTITION}
+    , {index_by_kafka_timestamp(PERSON_DISTINCT_ID_OVERRIDES_TABLE)}
+    """,
+)
```

Comment:
FYI: I did already add the data tables with #20326 — but not the Kafka ones yet. That was before I knew that the Kafka tables would be manipulated here. Sorry in advance for the merge conflicts. 😞

Reply:
I can rebase and figure it out.

Reply:
Kafka tables are mostly needed for testing (as we used to detach them so they had to be created to test that). But I have since removed that, so it should be possible to test without kafka tables and just rebase on `master`.

Reply:
Ah, somehow I completely over that they were removed here. 😬  Sorry about that -- thanks for clarifying! Not sure what I was looking at/thinking.